components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Utils/MethodCallStatusExeptionModel.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.OpcUa.Edge.Publisher.Tests.Engine {
+    using Newtonsoft.Json;
+    using System.Runtime.Serialization;
+    using static Microsoft.Azure.IIoT.Serializers.NewtonSoft.NewtonSoftJsonSerializer;
+
+    /// <summary>
+    /// Method call exception model.
+    /// </summary>
+    [DataContract]
+    public class MethodCallStatusExceptionModel {
+
+        /// <summary>
+        /// Exception message.
+        /// </summary>
+        [DataMember(Name = "Message", Order = 0,
+            EmitDefaultValue = true)]
+        public string Message { get; set; }
+
+        /// <summary>
+        /// Description.
```

Comment:
Let's change `Description` to something like `Details of the exception.`.

Reply:
changed